keel-sql/src/main/resources/db/changelog/20200522-standardize-timestamps.yml

```diff
@@ -1,39 +1,153 @@
 databaseChangeLog:
 - changeSet:
-    id: event-uid-timestamp-index
+    id: standardize-timestamps-agent_lock
     author: fletch
     changes:
+    - addColumn:
+        tableName: agent_lock
+        columns:
+        - column:
+            name: expiry_new
+            type: timestamp(3)
+            constraints:
+            - nullable: false
```

Comment:
Just to check my understanding: if you create a non-nullable timestamp column, MySQL will default the values to the current timestamp. Is that correct?

Reply:
correct.